README.md

```diff
@@ -16,11 +16,11 @@
 
 The Pi-hole® is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content without installing any client-side software.
 
-- **Easy-to-install**: our versatile installer walks you through the process and takes less than ten minutes
+- **Easy-to-install**: our versatile installer walks you through the installation process that takes less than ten minutes
```

Comment:
```suggestion
- **Easy-to-install**: our versatile installer walks you through the installation process, and takes less than ten minutes
```

Open for discussion on this one re: comma or not, but I think `and takes less...` sounds better than `that takes less...`

Reply:
I agree with you about `and takes ...`, but I think the comma should be removed.

I read this line like two sentences, linked by the `and`:
"**our versatile installer** walks you through the installation process and **our versatile installer** takes less than ten minutes".

Reply:
@Rauf00 I'm still uneasy on this sentence - something is not sitting right with me. It might even just need some new copy - the word versatile in this context has always read weird to me.

maybe something like:
our dialogs walk you through our simple installation process in less than ten minutes
our easy to follow installer gets you up and running in under ten minutes

Any more suggestions/thoughts from @pi-hole/core-maintainers ?

Reply:
@PromoFaux I agree. "Our dialogs walk you through our simple installation process in less than ten minutes" sounds good!

Reply:
@PromoFaux updated README.md as per your request.